.env.beta.mainnet

```diff
@@ -15,3 +15,4 @@ EXPLORER_URL=https://explorer.bitcoinvault.global
 SENTRY_DSN=https://dc67fd6d5c2949f2a6853e60eb69d899@o429172.ingest.sentry.io/5375289
 
 EMAIL_NOTIFICATIONS_API=https://btcv-notifcations-email.rnd.land/api
+AIRDROP_API = ""
```

Comment:
I know I said `AIRDROP_API = ""` but to keep consistency let's have it without whitespaces between `=` 😀

Reply:
🤣